source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,132 @@
+.. _scala-single-field-index:
+
+====================
+Single Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+Single field indexes are indexes with a reference to a single field of a
+document in a collection. These indexes improve single field query and
+sort performance. They also support :manual:`TTL Indexes </core/index-ttl>`
+that automatically remove documents from a collection after a certain
+amount of time or at a specified clock time.
+
+When creating a single-field index, you must specify the following
+details:
+
+- The field on which to create the index
+- The sort order for the indexed values as either ascending or
+  descending
+
+.. note::
+
+   The default ``_id_`` index is an example of a single-field index.
+   This index is automatically created on the ``_id`` field when a new
+   collection is created.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the
+``sample_mflix`` database from the :atlas:`Atlas sample datasets
+</sample-data>`. To learn how to create a free MongoDB Atlas cluster and
+load the sample datasets, see the :atlas:`Get Started with Atlas
+</getting-started>` guide.
+
+Create Single-Field Index
+-------------------------
+
+You must include the following import statements in your file to run the 
+following examples:
```

Comment:
Since the examples aren't directly following this sentence, and because this sentence structure has "the following" repeated, I think restructuring would be a little more clear:
```suggestion
To run the examples in this guide, you must include the following import statements in your file:
```